<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffChecker</class>
 <widget class="QWidget" name="DiffChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1137</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Diff Checker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #282A36;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="rightMargin">
        <number>500</number>
       </property>
       <item>
        <widget class="QPushButton" name="openFile1_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: #F8F8F2;
	border: 1px solid rgba(255,255,255,40);
	border-radius:7px;
	background-color:RGB(98, 114, 164);
	width: 100;
	height: 30;
}
QPushButton:hover{
	background-color:rgba(98, 114, 164,30);
}
QPushButton:pressed{
	background-color:rgba(98, 114, 164,70);
}</string>
         </property>
         <property name="text">
          <string>Open File 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openFile2_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: #F8F8F2;
	border: 1px solid rgba(255,255,255,40);
	border-radius:7px;
	background-color:RGB(98, 114, 164);
	width: 100;
	height: 30;
}
QPushButton:hover{
	background-color:rgba(98, 114, 164,30);
}
QPushButton:pressed{
	background-color:rgba(98, 114, 164,70);
}</string>
         </property>
         <property name="text">
          <string>Open File 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: #F8F8F2;
	border: 1px solid rgba(255,255,255,40);
	border-radius:7px;
	background-color:RGB(98, 114, 164);
	width: 100;
	height: 30;
}
QPushButton:hover{
	background-color:rgba(98, 114, 164,30);
}
QPushButton:pressed{
	background-color:rgba(98, 114, 164,70);
}</string>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: #F8F8F2;
	border: 1px solid rgba(255,255,255,40);
	border-radius:7px;
	background-color:RGB(98, 114, 164);
	width: 100;
	height: 30;
}
QPushButton:hover{
	background-color:rgba(98, 114, 164,30);
}
QPushButton:pressed{
	background-color:rgba(98, 114, 164,70);
}</string>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="file1_form"/>
       </item>
       <item>
        <widget class="QTextEdit" name="file2_form"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="diff_view_form">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
